src/composables/useFlawCvssScores.ts

```diff
@@ -48,10 +49,17 @@ export function useFlawCvssScores(flaw: Ref<ZodFlawType>) {
 
   const highlightedNvdCvss3String = computed(() => {
     const result = [];
-    const nvdCvssValue = nvdCvss3String.value;
-    const cvssValue = rhCvss3String.value;
+    const nvdCvssValue = flawNvdCvss3.value?.vector || '-';
+    const cvssValue = flawRhCvss3.value?.vector || '-';
     const maxLength = Math.max(nvdCvssValue.length, cvssValue.length);
 
+    if (formatScore(flawNvdCvss3.value?.score) !== formatScore(flawRhCvss3.value?.score)) {
```

Comment:
I moved the score out of the check since `10.0` and `7.0` have different number of characters (4 vs 3) which made the whole vector glow red as if everything was different.

|Before|After|
|---|---|
|![image](https://github.com/RedHatProductSecurity/osim/assets/4268580/572e1f46-8c3d-4e6c-8a81-efea8000e960)|![image](https://github.com/RedHatProductSecurity/osim/assets/4268580/a969eea3-3bb4-43d4-9add-26d3a9b73b43)|